packages/transcription/src/whisper/transcriber/ctranslate2-transcriber.ts

```diff
@@ -0,0 +1,51 @@
+import { $ } from 'execa'
+import { buildSUUID } from '@peertube/peertube-node-utils'
+import { lstat } from 'node:fs/promises'
+import { OpenaiTranscriber, WhisperTranscribeArgs } from './openai-transcriber.js'
+import { TranscriptFile } from '../../transcript/index.js'
+import { WhisperBuiltinModel } from '../whisper-builtin-model.js'
+import assert from 'node:assert'
+
+export class Ctranslate2Transcriber extends OpenaiTranscriber {
+  async transcribe ({
+    mediaFilePath,
+    model = new WhisperBuiltinModel('tiny'),
+    language,
+    format = 'vtt',
+    runId = buildSUUID()
+  }: WhisperTranscribeArgs): Promise<TranscriptFile> {
+    this.assertLanguageDetectionAvailable(language)
+
+    // Shall we run the command with `{ shell: true }` to get the same error as in sh ?
```

Comment:
Can you explain this comment? I'm not sure to understand it

Reply:
This can be safely removed, I was looking for a way to get the same exact output as if I was executing the command in my shell, but it probably has  too many drawbacks :
- https://github.com/sindresorhus/execa/issues/455
-  https://github.com/sindresorhus/execa/blob/main/docs/shell.md#os-specific-shell